tools/depmod.c

```diff
@@ -1066,6 +1066,10 @@ static int depmod_module_add(struct depmod *depmod, struct kmod_module *kmod)
 		size_t uncrelpathlen = lastslash - mod->relpath + modnamesz +
 				       strlen(KMOD_EXTENSION_UNCOMPRESSED);
 		mod->uncrelpath = memdup(mod->relpath, uncrelpathlen + 1);
+		if (mod->uncrelpath == NULL) {
+			err = -ENOMEM;
```

Comment:
missing a hash_del() like in the error path below?

Reply:
Indeed. Thank you for your careful review!